Tests/SentryTests/SentryClientTests.swift

```diff
@@ -28,74 +37,122 @@ class SentryClientTest: XCTestCase {
                 options.enabled = false
             })
         }
+    }
 
-        func assertEventNotSent(eventId: String?) {
-            XCTAssertNil(transport.lastSentEvent)
-            XCTAssertNil(eventId)
-        }
-
-        func assertLastSentEvent(assert: (Event) -> Void) {
-            if let lastSentEvent = transport.lastSentEvent {
-                assert(lastSentEvent)
-            } else {
-                XCTFail("LastSentEvent must not be nil")
-            }
-        }
-        
-        func assertLastSentEnvelope(assert: (SentryEnvelope) -> Void) {
-            if let lastSentEnvelope = transport.lastSentEnvelope {
-                assert(lastSentEnvelope)
-            } else {
-                XCTFail("LastSentEnvelope must not be nil")
-            }
+    private let error = NSError(domain: "domain", code: 0, userInfo: [NSLocalizedDescriptionKey: "Object does not exist"])
+    
+    private let exception = NSException(name: NSExceptionName("My Custom exception"), reason: "User clicked the button", userInfo: nil)
+    
+    private let environment = "Environment"
+    private var scope: Scope {
+        get {
+            let scope = Scope()
+            scope.setEnvironment(environment)
+            return scope
         }
     }
-
+    
     private let message = "message"
     private var fixture: Fixture!
 
     override func setUp() {
         super.setUp()
         fixture = Fixture()
     }
-
+    
     func testCaptureMessage() {
+        let eventId = fixture.getSut().capture(message: message, scope: nil)
+
+        XCTAssertNotNil(eventId)
```

Comment:
M: on Android we never return `nil`, but always an empty `UUID`, so people don't need to NPE check, is it the same for Cocoa? if so, a `not equal` assert would be better

Reply:
This is already there for a longer time. I also like the approach of not using `nil`, but I don't want to change this in this PR, because it would be a breaking change.